src/innsending/api.ts

```diff
@@ -11,3 +12,23 @@ export const sendInnSøknad = (søknad: object) => {
       return response.data;
     });
 };
+
+export const mapBarnTilEntenIdentEllerFødselsdato = (
+  barn: IBarn[]
+): IBarn[] => {
+  const filtrerteBarn = barn.map((barn) => {
+    if (barn.lagtTil) {
+      const endretBarn = barn;
+      if (endretBarn.ident.verdi === '') delete endretBarn.ident;
+      if (
+        endretBarn.fødselsdato &&
+        !endretBarn.fødselsdato.hasOwnProperty('verdi')
+      )
+        delete endretBarn.fødselsdato;
+      return endretBarn;
+    } else {
+      return barn;
+    }
+  });
+  return filtrerteBarn;
+};
```

Comment:
Hva har skjedd her?

Reply:
Dette var fordi jag bruke My sin branch som grunn, så dette var fra en av hennes commits :P 

> Ser du har tatt utgangspunkt i PR til My som hun ikke helt får til å virke - da har du to valg;
> Gjøre på nytt utfra master eller sørge for at hverken My eller du squasher før merge (og synkronisere evt. ting som fjernes)

Brukte den fordi master ikke virket med siste versjon av soknad-api, og jag fikk hennes til å virke fint :)

Reply:
Har gjort noen endringer i mappingsfunksjonen :) I stedet for å slette propertyen, så blir fødselsdato satt til en tom streng hvis den er undefined. Du må gjerne teste og sjekke om det fremdeles funker med de nye endringene mine :D